lib/Backend/LowerMDShared.cpp

```diff
@@ -2060,6 +2060,7 @@ LowererMD::LoadFunctionObjectOpnd(IR::Instr *instr, IR::Opnd *&functionObjOpnd)
         instr->InsertBefore(mov1);
         functionObjOpnd = mov1->GetDst()->AsRegOpnd();
         instrPrev = mov1;
+        instr->m_func->SetHasImplicitParamLoad();
```

Comment:
>SetHasImplicitParamLoad [](start = 23, length = 23)

You might want to add this setter in arm equivalent API for completeness, though you are not really optimizing for ARM